cves/kernel/CVE-2014-3183.yml

```diff
@@ -184,8 +222,11 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  answer: no
+  note: |
+    This is a implementation bug, when trying to limit the size of 
```

Comment:
I would suggest to add where you have looked for any mentions of violation of specifications then state this statement

Reply:
This is a slip (using < instead of >), so I don't see any possibilities that this is a violation of specs